conf/modules.config

```
@@ -54,5 +54,28 @@ process {
         ext.prefix = {"${meta.id}.trgt.sorted"}
     }
 
+    withName: 'TABIX_BGZIP' {
+        ext.prefix = {"${meta.id}.sv"}
+    }
+
+    withName: '.*:BAM_SV_VARIANT_CALLING:BCFTOOLS_INDEX' {
+        ext.prefix = { "${meta.id}.sv" }
+    }
 
+    withName: 'HIPHASE_SV' {
+        ext.prefix = {"${meta.id}.sv"}
+    }
+
+    withName: 'HIPHASE_SNP' {
+        ext.prefix = {"${meta.id}.snv"}
+    }
+
+
+    withName: 'GATK4_HAPLOTYPECALLER' {
+        ext.prefix = {"${meta.id}.snv"}
+    }
+
+    withName: 'DEEPVARIANT' {
+        ext.prefix = {"${meta.id}.snv"}
```

Maybe it makes sense to add the corresponding tool used for calling here
```suggestion
        ext.prefix = {"${meta.id}.deepvariant.snv"}
```

and similar with the others. I am not sure about the output paths, so if they are sorted into folders this might not be necessary but that makes it easier to identify later on what was actually done here.

---

They should be in different folders (if the user uses deepvariant the file will be in the deepvariant folder, if they use gatk4 it will be in the gatk4 folde) - so I think i'll stick with using just the snv as the name for now!. Thanks!